src/timings/Timings.php

```diff
@@ -299,4 +306,46 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 
 		return self::$eventHandlers[$event][$handlerName];
 	}
+
+	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task, string $group = self::GROUP_BREAKDOWN) : TimingsHandler{
+		$taskClass = $task::class;
+		if(!isset(self::$asyncTasksProgressUpdate[$taskClass])){
+			self::init();
+			self::$asyncTasksProgressUpdate[$taskClass] = new TimingsHandler(
+				"AsyncTask - " . self::shortenCoreClassName($taskClass, "pocketmine\\") . "::onProgressUpdate()",
```

Comment:
This isn't actually measuring `onProgressUpdate()`, as it won't be called if there are no progress updates available.